src/ngx_http_lua_shdict.c

```diff
@@ -531,6 +581,14 @@ ngx_http_lua_shdict_get_helper(lua_State *L, int get_stale)
         lua_pushboolean(L, c ? 1 : 0);
         break;
 
+    case LUA_TTABLE:
```

Comment:
I think using `LUA_TTABLE` here is inappropriate since a Lua table can be either an array or a hash table or both at the same time. We'll eventually have hash-table value type in shdict. So better invent our own type constants instead of abusing Lua's types?